helpers/eventwebhook/eventwebhook.go

```diff
@@ -19,7 +19,7 @@ const (
 
 // Settings ...
 type Settings struct {
-	Enable *bool `json:"enabled,omitempty"`
+	SecureWebhookEnable *bool `json:"enabled,omitempty"`
```

Comment:
could we call this `EnableSignedWebhookRequests` instead?

Reply:
Is it a request?

Reply:
that's how the UI presents it; when we post, they read the POST request (on their end, it is considered a request)

Reply:
but none of the function is doing post(or any other http method) request in this file.